frontend/app/views/fragments/bundle/bundleButton.scala.html

```diff
@@ -0,0 +1,25 @@
+
+@(buttonLabel: String, target: String, dataLink: String, chevron: Boolean = false)
+
+<span class="bundle-button">
+    <a data-link-name="@dataLink" class="link-button" href="@target">
+        <div class="bundle-button__content">
+            <p>@buttonLabel</p>
+            <span class="bundle-button__content__arrow">
+                @if(chevron){
+                    <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Shouldn't these SVGs be added as separate files (eg https://github.com/guardian/membership-frontend/blob/master/frontend/assets/images/inline-svgs/raw/chevron-right.svg) and accessed with `fragments.inlineIcon()` as an SVG sprite?